### PR TITLE
getNFTSales updates

### PR DESCRIPTION
**Pull request by alex-miao**

### CODE REVIEW

File: nft/nfts.yaml [hunk 1]
```diff
@@ -740,27 +739,21 @@ paths:
...
-                        protocolFeeCurrency:
-                          type: string
-                          description: String - The token used to pay the marketplace in the sale.
+                          description: String - Whether the taker in the trade was the buyer or the seller.
+                          enum:
+                            - BUYER
+                            - SELLER
+                        sellerFee:
+                          type: object
+                          description: The amount, token symbol, and decimals of the token of the payment from buyer to seller
```
Comment (elanh) [inline nft/nfts.yaml:750]: is seller fee the right name here or is sellerPrice better? fee makes me think it's an additional charge on top of the price of the NFT, not the actual total cost of the NFT which is what this seems like it is.

Comment (elanh) [reply nft/nfts.yaml:750]: ahh i see below there are three components into the total cost - do we also want to add a variable for total cost which is the sum of these three (marketplaceFee + royaltyFee + sellerFee)

or does the royaltyFee come out of the sellerFee?

Comment (alex-miao) [reply nft/nfts.yaml:750]: The total price is the sum of these three values, but it's kind of annoying that in Seaport the fees don't have to be the same token. I would like to have something like `totalUSDPrice` in the future, but for now could add a `totalPrice` field that is null if the fee tokens differ.

Comment (elanh) [reply nft/nfts.yaml:750]: totalUSDPrice makes sense in the future

**Review by elanh — APPROVED** [review]

**Pull request merged**